deploy/roles/sas/tasks/main.yml

```diff
@@ -37,11 +43,16 @@
     line: 'JVM_OPTS="{{ java_heap_size }}"'
     backup: yes
   when: install_sas
```

Comment:
@rioh I think we should remove the `when: install_sas` conditional here, since it may be the case that we want to change the JVM heap size, but not run the full install script.